src/match_stream_client.erl

```diff
@@ -66,8 +66,9 @@ wait_for_socket({socket_ready, Socket}, State) ->
     end,
   ?DEBUG("Client on ~p connected~n", [PeerPort]),
   ok = inet:setopts(Socket, [{active, once}, {packet, 0}, binary]),
-  {next_state, wait_for_params, State#state{socket   = Socket,
-                                            peerport = PeerPort}};
+  NewState = State#state{socket = Socket, peerport = PeerPort},
+  ok = tcp_send(Socket, "Welcome to Match Stream.\nTo watch the current game use: V:2:<your-name>\n", NewState),
+  {next_state, wait_for_params, NewState};
 wait_for_socket(timeout, State) ->
   ?WARN("Socket wasn't set fast enough for client on ~p~n", [State#state.peerport]),
```

Comment:
Line 73 is too long:   ?WARN("Socket wasn't set fast enough for client on ~p~n", [State#state.peerport]),.